### DISCUSSION TITLE
Discussion about datafusion meetups

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#10347] [Ideas]
Let's coordinate face to face meetups here

Other meetup discussions:
* SF Bay Area: https://github.com/apache/datafusion/discussions/10340
* Europe: https://github.com/apache/datafusion/discussions/10342
* China: https://github.com/apache/datafusion/discussions/10341
* US East Coast: https://github.com/apache/datafusion/discussions/10343
* Seattle: https://github.com/apache/datafusion/discussions/10348
* Austrailia: https://github.com/apache/datafusion/discussions/10349
* India: https://github.com/apache/datafusion/discussions/10351
* Midwest USA: https://github.com/apache/datafusion/discussions/11568
* Amsterdam: https://github.com/apache/datafusion/discussions/12988

**Comment by advancedxy** [comment]
Hmm, I think the China link is wrong, it should be https://github.com/apache/datafusion/discussions/10341 instead.

**Reply by alamb** [reply]
Thank you -- fixed!

**Comment by timsaucer** [comment]
Added https://github.com/apache/datafusion/discussions/11568 for Midwest USA

**Comment by kavirajk** [comment]
I spoke to one of the organisers(@Dtenwolde) of [The Dutch Seminar on Data Systems Design](https://dsdsd.da.cwi.nl/) group. They are very much interested to organise DataFusion (in-person) meetup at Amsterdam. 

Side-note: @alamb already gave a (virtual)[ talk](https://dsdsd.da.cwi.nl/past_talks/post_talks/Andrew-Lamb/) about InfluxDB IOx a year ago there.

how do we proceed further?

**Reply by alamb** [reply]
Hi @kavirajk ! that is great news. I think in order to do a meetup all we need to do is find someone(s) who want to organize it (aka find a meeting space, do the organizing).  I think that is about it -- I am happy to share what we have done for other meetups (agenda, advertisements, etc) to help

It is probably a bit far away but one option is to organize such a meetup around the time of CIDR 2025 (when some non trivial number of people might already be in Amsterdam) https://www.cidrdb.org/cidr2025/ 🤔

**Reply by kavirajk** [reply]
Thanks @alamb 👍 

> to organize such a meetup around the time of CIDR 2025

Interesting option :) will wait to hear what @Dtenwolde thinks

**Reply by Dtenwolde** [reply]
Hi there, 

Organizing the meetup as a satellite event of CIDR makes sense since many database enthusiasts will be in Amsterdam around that time. CIDR traditionally ends with lunch on Wednesday, but in 2023 there was the [Amsterdam Data Science meetup in the afternoon](https://www.cidrdb.org/cidr2023/program.html#:~:text=Drinks%20%26%20Data%3A%20CIDR%20meets%20ADS). I am not involved in the organisation of CIDR, so I don't know if there's anything already planned for CIDR2025, but perhaps it is a possibility to host the meetup in the Wednesday afternoon?

Possibly folks from DuckDB Labs are also interested in helping out (if that is something you want). 

With kind regards, 
Daniel

**Reply by kavirajk** [reply]
> I am not involved in the organisation of CIDR, so I don't know if there's anything already planned for CIDR2025, but perhaps it is a possibility to host the meetup in the Wednesday afternoon?

I don’t have a strong preference, but my two cents is that it would be great to have it in the evening, as people from the industry with full-time jobs (like me) will have no problem joining :)

**Reply by alamb** [reply]
FWIW we planned a datafusion meetup in Austin to coincide with a conference (Data Council) and it seemed to work well. More details on https://github.com/apache/datafusion/discussions/8522

**Reply by kavirajk** [reply]
Happy to see this going in a good direction :)

It may be a bit too early to plan everything for the meetup that is going to happen in January 2025. I will leave it up to @alamb and @Dtenwolde to figure out the details. I’m happy to help organize and make it happen :)

We can create a separate discussion similar to https://github.com/apache/datafusion/discussions/8522 once we have more details (closer to November or December may be).

**Reply by alamb** [reply]
I created https://github.com/apache/datafusion/discussions/12988 to discuss this more. Looking forward to it!
